### ISSUE TITLE
LoadError: cannot load such file -- 2.5/ffi_c

### ISSUE THREAD

**Issue by marvindanig** [rails/rails#38112]
I have a rails 5.2.0 set up on my macos catalina with rbenv 1.1.2/ruby 2.5.1. At the root of my app I get the following error:
 
```
$ rails -T
rails aborted!
LoadError: dlopen(/Users/sa/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/ffi-1.11.1/lib/ffi_c.bundle, 9): Symbol not found: _ffi_type_double
  Referenced from: /Users/sa/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/ffi-1.11.1/lib/ffi_c.bundle
  Expected in: flat namespace
 in /Users/sa/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/ffi-1.11.1/lib/ffi_c.bundle - /Users/sa/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/ffi-1.11.1/lib/ffi_c.bundle
/Users/sa/Projects/ror/bubblin.io/bubblin.io/config/application.rb:7:in `<top (required)>'
/Users/sa/Projects/ror/bubblin.io/bubblin.io/Rakefile:1:in `<top (required)>'
bin/rails:4:in `<main>'

Caused by:
LoadError: cannot load such file -- 2.5/ffi_c
/Users/sa/Projects/ror/bubblin.io/bubblin.io/config/application.rb:7:in `<top (required)>'
/Users/sa/Projects/ror/bubblin.io/bubblin.io/Rakefile:1:in `<top (required)>'
bin/rails:4:in `<main>'

Caused by:
Bootsnap::LoadPathCache::FallbackScan:

(See full trace by running task with --trace)
```

I can provide the full trace but just to know I reinstalled everything from scratch and my xcode-command-line tools/gcc are up-to-date. Is this something that has come up/been solved before?

**Comment by eugeneius** [comment]
Like https://github.com/rails/rails/issues/38110, this is a problem with your environment, not with Rails.

Please use [Stack Overflow](https://stackoverflow.com/questions/tagged/ruby-on-rails) or the [rubyonrails-talk](https://groups.google.com/forum/#!forum/rubyonrails-talk) mailing list for questions, where a wider community will be able to help you. We reserve the issue tracker for bugs only.

**Comment by marvindanig** [comment]
I have posted a few questions on Stack Overflow before, and quite a few of them went unanswered. I posted this issue over [there](https://stackoverflow.com/questions/59512555/rails-t-rails-server-puma-is-failing) as well. Somehow the wider community doesn't appear to be as engaged as it used to be earlier… and it might just make sense to no longer restrict issue tracker here on Github for bugs only. 

Just sharing a tiny feedback but then what do I know. New year ahead!
